x-pack/plugins/spaces/public/management/components/solution_view/solution_view.tsx

```diff
@@ -112,25 +107,40 @@ export const SolutionView: FunctionComponent<Props> = ({
   sectionTitle,
 }) => {
   const { euiTheme } = useEuiTheme();
+  const showClassicDefaultViewCallout = isEditing && space.solution == null;
 
   return (
     <SectionPanel title={sectionTitle} dataTestSubj="navigationPanel">
       <EuiFlexGroup>
         <EuiFlexItem>
           <EuiTitle size="xs">
-            <h3>
-              <FormattedMessage
-                id="xpack.spaces.management.manageSpacePage.setSolutionViewMessage"
-                defaultMessage="Set solution view"
-              />
-            </h3>
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <h3>
+                  <FormattedMessage
+                    id="xpack.spaces.management.manageSpacePage.setSolutionViewMessage"
+                    defaultMessage="Select solution view"
+                  />
+                </h3>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiBetaBadge
```

Comment:
If solution view is set:
<img width="1455" alt="image" src="https://github.com/user-attachments/assets/807465f9-c36d-4beb-bb97-5e097fa4e545">

If classic view is set:
<img width="1457" alt="image" src="https://github.com/user-attachments/assets/639d0d4d-3119-4d29-a518-41dfb60cf0af">